extensions/data-plane/README.md

```diff
@@ -0,0 +1,4 @@
+# Data Plane module
+
+A comprehensive description of the Data Plane can be
+found [here](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/463).
```

Comment:
would prefer to have the documentation as part of the repo (under version control; issues are not part of code checked out etc).

Reply:
Documentation imported into the repo.